automation/vars/default.yaml

```diff
@@ -224,3 +224,47 @@ vas:
           - name: edpm-nodeset-values
             src_file: values.yaml
         build_output: edpm.yaml
+
+  bgp:
+    stages:
+      - path: examples/dt/bgp/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60s
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/bgp/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait osctlplane controlplane --for condition=Ready
+            --timeout=30m
+        values:
+          - name: network-values
+            src_file: nncp/values.yaml
+        build_output: ../control-plane.yaml
```

Comment:
This should maybe just be like this [1]?

[1] https://github.com/openstack-k8s-operators/architecture/blob/3e0b37a46e4bb47f1fba2b0bcf2bc2662b956d65/automation/vars/uni01alpha.yaml#L39